lib/presenter/Chart.js

```diff
@@ -419,6 +425,10 @@ class Chart extends Events {
         var legendWidth = this.width - this.margins.left - this.margins.right - doubleLegendMargin;
         var legendHeight = this.legendHeight - doubleLegendMargin;
 
+        // Clamp width/height at 0 to avoid SVG errors
+        legendWidth = (legendWidth < 0) ? 0 : legendWidth;
+        legendHeight = (legendHeight < 0) ? 0 : legendHeight;
```

Comment:
This fixes all of those obnoxious console errors we see in tests all of the time: `remote console.log: Error: A negative value for rect <width> is not allowed`